app/(tabs)/expenses/index.tsx

```diff
@@ -46,11 +45,41 @@ export default function SettleScreen() {
     swipeableRows.map((row) => row.close())
   };
 
-  const renderItem = ({ item, index }: { item: Transaction; index: number }) => {
+  function getExpenses(expenseData: Expense[]): void{ 
+    if (!expenseListCmp(expenseData, data)){
+      setData(expenseData);
+    }
+  }
+
+  function getGroupMembers(_members: string[]): void{
+    if (_members != members){
+      setMembers(_members);
+    }
+  }
+
+  let userGroup = '';
+  gun.user(userPub).get('group').get('groupId').once((id: string) => {
+    userGroup = id;
+    console.log(userGroup)
+  });
+
+  let username = '';
+  gun.user(userPub).get('fullName').open((name: string) => {
```

Comment:
Just use once here instead of open, as there should only be a top level string returning anyway